.github/workflows/cli-tests.yaml

```diff
@@ -77,3 +77,18 @@ jobs:
           echo "PATH=$PATH" >> $GITHUB_ENV
       - name: Run tests
         run: go test -race -cover ./...
+
+  cli-cicd-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: ./go.mod
+          cache: true
+      - name: Build devbox
+        run: go install ./cmd/devbox
+      - name: Install nix and devbox packages
```

Comment:
Add comment that main reason we have this test is to ensure Devbox installs nix and works properly after installation.